vinyl/display.php

```diff
@@ -1,24 +1,29 @@
 <?php
 
-//link database to page 
-$db = new PDO('mysql:host=db; dbname=Vinyl', 'root', 'password');
-$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
-
-$query = $db->prepare("SELECT `record-name`, `artist-name`, `record-size` FROM `records`;");
-$query->execute();
+require_once "functions.php";
 
-$data = $query->fetchAll();
+$db = connectTodatabase(); 
+$data = fetchAlldatabase($db);
 
-require_once "functions.php";
 ?>
 
 <html>
```

Comment:
you may want to give a language property here (not essential!) e.g. <html lang="en">